src/mscorlib/model.xml

```diff
@@ -3556,21 +3516,6 @@
       <Member Name="get_Value"/>
       <Member Status="ImplRoot" Name="OnSerializing(System.Runtime.Serialization.StreamingContext)" />
     </Type>
-    <Type Name="System.LoaderOptimization">
-      <Member MemberType="Field" Name="DisallowBindings" />
-      <Member MemberType="Field" Name="DomainMask" />
-      <Member MemberType="Field" Name="MultiDomain" />
-      <Member MemberType="Field" Name="MultiDomainHost" />
-      <Member MemberType="Field" Name="NotSpecified" />
-      <Member MemberType="Field" Name="SingleDomain" />
-      <Member MemberType="Field" Name="value__" />
-    </Type>
-    <Type Name="System.LoaderOptimizationAttribute">
-      <Member Name="#ctor(System.Byte)" />
-      <Member Name="#ctor(System.LoaderOptimization)" />
-      <Member Name="get_Value" />
-      <Member MemberType="Property" Name="Value" />
-    </Type>
     <Type Name="System.LocalDataStoreSlot">
```

Comment:
LocalDataStoreSlot is implemented in corefx as well. Is this just the first pass of cleanup?